src/index.js

```diff
@@ -57,7 +38,7 @@ export default function brotli(options = {}) {
     },
     writeBundle: async (outputOptions, bundle) => {
       const compressCollection = []
-      const bundlesToCompress = Object.keys(bundle).filter(id => !isCompressed(bundle[id])).map(id => bundle[id].fileName)
+      const bundlesToCompress = Object.keys(bundle).map(id => bundle[id].fileName).filter(fileName => options.test.test(fileName))
```

Comment:
I feel it's still worth keeping magic bytes as they're a reliable mechanism for testing the contents of the file to see if it is already compressed.

Reply:
Updated to still include the isCompressed / magic bytes checks and updated README